### PR TITLE
 Highlight the lines base on total coverage

### PR DESCRIPTION
**Pull request by LinkaiQi**
Refer to Issue #33 

1. Show a number in front of each line represent the number of test that cover it.
2. Highlight the lines base on coverage. (Red -> uncovered line, Green-> covered line)

### CODE REVIEW

**Comment by LinkaiQi** [comment]
Attach a screenshot,

screen shot 2017-11-09 at 1 00 26 am

File: src/components/fileviewer.js [hunk 1]
```diff
@@ -64,14 +64,18 @@ export default class FileViewerContainer extends Component {
...
   parseTestsCoverage(data) {
     console.log(data.length);
-    var stat = []
+    var stat = [];
+
+    // initial list for each coverable line
+    if (data.length > 0) {
+      const fileCov = data[0].source.file;
+      fileCov.covered.concat(fileCov.uncovered).forEach(line => {
+        stat[line] = [];
```
Comment (klahnakoski) [inline src/components/fileviewer.js:73]: Interesting!

**Comment by klahnakoski** [comment]
You have a url I can use to view a file?

**Comment by klahnakoski** [comment]
nm, I used http://localhost:3000/file?revision=a0334f789772&path=/accessible/atk/Platform.cpp  just fine

File: src/components/fileviewer.js [hunk 2]
```diff
@@ -144,13 +149,26 @@ const Line = (props) => {
...
+  }
+
+  let nTests, coverage = "";
+  if (props.testsPerLines[props.lineNumber]) {
+    nTests = props.testsPerLines[props.lineNumber].length
+    if (nTests > 0) {
+      coverage = "hit";
+    } else {
+      coverage = "miss";
+    }
```
Comment (klahnakoski) [inline src/components/fileviewer.js:162]: For simple logic like this, try `coverage = (nTests > 0) ? "hit" : "miss";`

**Pull request merged**